packages/quantic/force-app/examples/main/lwc/exampleQuanticFacetManager/exampleQuanticFacetManager.html

```diff
@@ -20,6 +20,7 @@
           <c-quantic-timeframe unit="year"></c-quantic-timeframe>
         </c-quantic-timeframe-facet>
         <c-quantic-facet engine-id={engineId} field="objecttype" label="Type"></c-quantic-facet>
+        <div slot="itemTemplate" class="slds-m-around_xx-large"></div>
```

Comment:
What's that for?

Reply:
that is specifically to test the "itemTemplate" slot logic in the e2e test, you can see the describe on line 68:
**when an itemTemplate slot is provided**

Reply:
Ah it's the item wrapper ok